tests/wpt/mozilla/tests/css/perspective.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <title>CSS Test: parsing perspective property</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+</head>
+
+<body>
+    <div id="perspective0px" style="perspective: 0px"></div>
```

Comment:
Yes, that was the reason. When I reset the perspective property after the assert it works as expected. The undefiend for the 'none' keyword was my mistake.